sub-projects.md

```diff
@@ -0,0 +1,13 @@
+# ORAS Sub-Projects
+
+The ORAS Project represents the following sub-projects:
+
+| Sub-Project | Repositories | Sub-Project Maintainers |
+| - | - | - |
+| Org Governance | [governance](https://github.com/oras-project/community) | [CODEOWNERS](https://github.com/oras-project/community/blob/main/CODEOWNERS) | 
```

Comment:
```suggestion
| Org Governance | [governance](https://github.com/oras-project/community/blob/main/governance/GOVERNANCE.md) | [CODEOWNERS](https://github.com/oras-project/community/blob/main/CODEOWNERS) | 
```

Reply:
thanks @TerryHowe. I see you wanted to link to the community governance files. In this case, we're defining each sub-project and the associated repositories. 
The sub-project is "**Org Governance**", with the repo being "**oras-project/community**", rather than a link to the governance document. For instance, we should likely have some templates within the repo.